src/Page-Components/Article.jsx

```diff
@@ -34,20 +35,29 @@ const Article = () => {
       {if(!Object.keys(comments).length){
         return (
           <div className="articleInfo text-black font-bold"><p>No Comments...</p></div>
-        )
-      }
+        )}
       else{
         return (
           <ul>
           {comments.map((comment) => {return (
             <li key={comment.comment_id}><CommentCard comment={comment}/></li>)})
           }
           </ul>
-        )
-      }
+        )}
     }
   }
 
+  const upVote = (articleId) => {
+    fetchApiData.patch(`/articles/${articleId}`, {inc_votes:1}).then(({data}) => {
+      setArticle(data.updatedArticle)})
+  };
+
+  const downVote = (articleId) => {
+    fetchApiData.patch(`/articles/${articleId}`, {inc_votes:-1}).then(({data}) => {
+      setArticle(data.updatedArticle)})
+  };
+
```

Comment:
Nice work but we will need to handle this optimistically. Please increase or decrease the vote count displayed at the beginning of the function (before the patch request) and undo it if the catch block is reached. 

And if you have time make sure to do your api calls in another file and import the functions to keep things clean.

Thanks :)